### DISCUSSION TITLE
Tauri desktop app template

### DISCUSSION THREAD

**Discussion by erlend-sh** [freshframework/fresh#613] [Ideas]
Would love a tauri-fresh-template, like this:
https://github.com/probablykasper/tauri-svelte-template

Some prior art from nextjs+electron:
https://github.com/saltyshiomix/nextron
https://blog.logrocket.com/building-app-next-js-electron/

**Comment by Eudritch** [comment]
I made a tauri-SvelteKit template based on the svelte-template:
[https://github.com/Eudritch/clean-tauri-sveltekit](https://github.com/Eudritch/clean-tauri-sveltekit)

**Comment by erlend-sh** [comment]
Related: https://github.com/denoland/fresh/discussions/254
